mysql-test/suite/rpl/t/rpl_percona_bug995624.test

```diff
@@ -0,0 +1,46 @@
+source include/master-slave.inc;
```

Comment:
Please name rpl_percona_userstat, and add a
--echo #
--echo # Bug 995624 blah blah
--echo #
section. So that further replication-related userstat testcases could be added to the same file